playwright.config.ts

```diff
@@ -72,7 +72,7 @@ export default defineConfig({
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: 'yarn serve',
+    command: 'yarn start',
```

Comment:
Why this change, does it make a difference?

Would be slightly more comfortable testing against "serve" since that's closer to the prod setup, but I guess if it makes tests more reliable, I want that even more.

Reply:
No, it makes no difference from my experience. I will change this back in a follow up